src/App.js

```diff
@@ -1,16 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+  const [entries, setEntries] = useState(chatMessages);
+
+  const startLikeCount = entries.filter((entry) => entry.liked).length;
+
+  const [likeCount, setLikeCount] = useState(startLikeCount);
```

Comment:
Good use of state here, and I especially like how you created this constant for the startLikeCount. One small thing to note though is that we don't necessarily need the likeCount to be a state variable because contrary to popular belief, we won't be passing it among components. It is a variable that we only work with within this component, so we won't be passing it anywhere else. I will explain further in a later comment!